Classes/APmedicine.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="APmedicine.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+namespace Hw8.Classes
+{
+    using Hw8.Enums;
```

Comment:
Please create a separate folders for Models and Services, which operates those model classes